releases/net471/KnownIssues/514195-Targeting .NET Framework 4.7.1 will copy extra files to your bin directory.md

```diff
@@ -0,0 +1,24 @@
+# Targeting .NET Framework 4.7.1 will copy extra files to your bin directory
+
+## Symptoms
+
+When building an application that targets .NET Framework 4.7.1, the tooling will copy 12 assemblies into your bin directory if you consume one or more .NET Standard-based components.
+This will result in the bin output of your application to be larger and include more files.
+
+## Cause
+
+.NET types live in different assemblies depending on the framework you are running in. A simple example would be the class String, which lives in mscorlib.dll on .NET framework, in netstandard.dll when you target .NET Standard 2.0, in System.Runtime.dll when you target .NET Standard 1.x, and in System.Private.CoreLib.dll on .Net Core. In order to have a library targeting one framework to successfully load when running on a compatible one, we sometimes need runtime facades (with type-forwards) which help us re-route the references to the all of the types that live on different places.
```

Comment:
- framework you are running in --> .NET implementation you are running in
- would be --> is
- class String --> String class 
- on .NET framework --> on .NET Framework
- "In order to have a library targeting one framework to successfully load..." --> "For a library targeting one ,NET implementation to load successfully..."
- "we sometimes need runtime facades (with type-forwards) which help us re-route" --> runtime facades (with type-forwards) are ordinarily used to reroute..." (note that I replaced "sometimes" with "ordinarily" -- the change makes the following paragraph clearer. 
- on different --> in different